client/src/Components/Header.js

```diff
@@ -1,18 +1,27 @@
 import React from 'react'
 import styled from 'styled-components'
 
-const Name = () => {
-    return( <NameStyles>Give It Back It`s worthit </NameStyles>
+const HGIB = () => {
+    return( <Header>
+        <HgibStylesOne>Give It Back </HgibStylesOne>
+        <HgibStylesTwo>It`s worthit</HgibStylesTwo>
+        <hr></hr>
+        </Header>
     );
 };
 
-const NameStyles = styled.h1`
-height: 75px;
+const HgibStylesOne = styled.h1`
 justify-content: center;
-margin: 5px; 
-border: 3px black solid;
-align-items: center;
-color: white;
-background-color: green
+text-align: center;
 `;
-export default Name;
\ No newline at end of file
+
+const HgibStylesTwo = styled.h1`
```

Comment:
We do not use more than one h1 per page, for semantic reasons. You can nest spans in h1 if you need different styling for your heading.

Reply:
Could you please describe what nest spans are?